sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala

```diff
@@ -286,6 +286,8 @@ abstract class Expression extends TreeNode[Expression] {
   override def simpleStringWithNodeId(): String = {
     throw new UnsupportedOperationException(s"$nodeName does not implement simpleStringWithNodeId")
   }
+
+  val FUNC_ALIAS = TreeNodeTag[String]("functionAliasName")
```

Comment:
We can define it in `object FunctionRegistry`. This is kind of a map key, and we don't need to create a different instance for each expression.